merf/merf.py

```diff
@@ -242,6 +245,17 @@ def fit(self, X, Z, clusters, y):
             logger.info("GLL is {} at iteration {}.".format(gll, iteration))
             self.gll_history.append(gll)
 
+            if self.gll_early_stop_threshold is not None:
+                if not iter_begun:
```

Comment:
@cunningjames I am looking at this really really late, I know. Apologies. It has been hard maintaining an open-source packages while starting a company, but getting to it now :). 

In any case, this code is fairly straight forward -- but I don't understand this "iter_begun" flag. What is it for? It seems to be superfluous -- like if the early stop threshold is "on" it just sets it to True -- but is it ever used afterwards?